content/2021-11-19-brigade-2-cuts-first-release-candidate.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Brigade 2 Cuts First Release Candidate"
+slug: "2021-11-19-brigade-2-cuts-first-release-candidate"
+description: "Brigade 2 has cut its first release candidate"
+date: "2021-11-19"
+author: "Kent Rancourt"
+summary: Brigade 2 has cut its first release candidate.
+---
+
+We have exciting news, Brigadiers!
+
+Yesterday afternoon, maintainers cut the first Brigade 2 release candidate. This release candidate ([v2.0.0-rc.1](https://github.com/brigadecore/brigade/releases/tag/v2.0.0-rc.1)) also marks Brigade 2's transition from serving the v2 beta API to serving the _stable_ v2 API. With this new level of stability, there's never been a better time to get started with Brigade 2, so please do consider heading over to our recently refreshed [QuickStart](https://v2--brigade-docs.netlify.app/intro/quickstart/) and taking us for a spin.
+
+The transition to the stable API means previous versions of gateways and other peripherals are not compatible with this release candidate, but as of this morning, we have _also_ transitioned every one of those projects to the stable API and cut a beta release of each. (They each currently remain at a beta level of maturity despite Brigade 2 itself having advanced to a release candidate.)
+
+Barring any major surprises, we anticipate the only PRs merged between now and December 1 will be release-oriented chores, so we're targeting that date for our GA release -- just on time for the holiday season!
+
+Following the Brigade 2 GA release, maintainers will begin putting the finishing touches on all our peripherals and will release each of those individually as they reach maturity, with the GitHub gateway almost certainly being the first.
```

Comment:
Do readers know what a peripheral is? Do you want to say something like "peripherals (foo, bar, bat, etc)"?

Reply:
Am I wrong to assume it's a commonly understood word? The first usage here (a couple paragraphs up) says "gateway and other peripherals." I _think_ it's understood to be inclusive of gateways as well as integrations that are _not_ gateways...

We could start favoring the term "integration," but I do like that peripheral sounds like what it is-- something we built as an accessory of sorts.

lmk what you think.

Reply:
People may understand the general concept, but they would probably like a list of which items you're talking about specifically. Maybe navigating over to https://github.com/brigadecore/ and picking a few to highlight would be helpful in making sure people know which items to go look at?

Reply:
I see what you're saying. I'm going to include an example or two to lend some clarity to how I'm using the term.

Reply:
Fixed in https://github.com/brigadecore/blog/commit/021e94694b4438507085c17e00f00f3a260c001e